src/templates/templates.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageType>Template</PackageType>
+    <PackageId>Microsoft.Android.Templates</PackageId>
+    <Title>.NET Android Templates</Title>
+    <Authors>Microsoft</Authors>
+    <Description>Templates for Android platforms.</Description>
+    <IncludeContentInPack>true</IncludeContentInPack>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <ContentTargetFolders>content</ContentTargetFolders>
+    <OutputPath>..\..\bin\Build$(Configuration)\nupkgs\</OutputPath>
+  </PropertyGroup>
+
+  <Import Project="..\..\Configuration.props" />
+
+  <ItemGroup>
+    <Content Include="**\*" Exclude="**\bin\**;**\obj\**" />
```

Comment:
Shouldn't this use `Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)"`?

Reply:
Actually we can't use these in a template pack...

`$(DefaultItemExcludes)` excludes `.csproj` files, which are needed in templates.

https://github.com/dotnet/sdk/blob/44f381b62d466565639d51847c9127afbe7062a9/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L33

`$(DefaultExcludesInProjectFolder)` will exclude folders that start with a `.`, so we can't use it because of `.templateconfig/template.json`.